cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,16 @@
+/// <reference types="cypress" />
+
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
 
+    cy.visit('user/register');
+    cy.get('input[placeholder="Username"]').type(user.username);
+    cy.get('input[type="email"]').type(user.email);
+    cy.get('input[type="password"]').type(user.password);
+    cy.get('button[type="submit"]').click();
+    cy.get('.nav-link').should('contain', user.username.toLowerCase());
```

Comment:
This line checks if the username appears in the navigation bar, which is good. However, it would be better if you also check other user details, such as email, to ensure that the user was registered correctly.